.github/workflows/sbom-convert.yml

```diff
@@ -0,0 +1,23 @@
+name: Workflow for converting CycloneDX SBOM files to SPDX format
+
+on:
+  workflow_call:
+
+jobs:
+  convert:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/download-artifact@master
+        with:
+          name: cyclonedx-json
+          path: ./cyclonedx-sbom
+      - name: Convert SBOM
+        run: |
+          wget https://github.com/CycloneDX/cyclonedx-cli/releases/download/v0.24.2/cyclonedx-linux-x64
```

Comment:
is this just the contents of the script below?  is it possible to just run that script?